app/uk/gov/hmrc/contactadvisors/controllers/EmailController.scala

```diff
@@ -50,7 +50,9 @@ class EmailController @Inject()(
     {
       implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromHeadersAndSession(request.headers)
       authorised(predicates.foldLeft[Predicate](AuthProviders(PrivilegedApplication))((b, a) => b and a)) {
-        request.body.asJson.fold(Future.successful(BadRequest("""{"error": "invalid payload"}""")))((json: JsValue) => emailService.doSendEmail(json))
+        request.body.asJson
+          .flatMap(_.asOpt[JsObject])
+          .fold(Future.successful(BadRequest("""{"error": "invalid payload"}""")))((jsobj: JsObject) => emailService.doSendEmail(jsobj))
```

Comment:
do we want to return a more specific error of what is wrong? 
Also maybe log the exception as kibana error log message?

Reply:
it is not an error of the microservice but the client error. If a careless client sends a lot of invalid request we will receive PagerDuty alerts.